e2e/cypress/integration/messaging/visual_verification_of_tooltips_on_top_nav_channel_icons_posts_spec.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Messaging', () => {
+    let testTeam;
+
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+
```

Comment:
Here, please post message as sysadmin so you'll have date separator, and then later on show new message line (by marking post by sysadmin as unread)

```javascript
import {getAdminAccount} from '../../support/env';
...

            const daysAgo = Cypress.moment().subtract(10, 'days').valueOf();
            cy.postMessageAs({sender: admin, message: `Hello from admin 10 days ago`, channelId: channel.id, createAt: daysAgo});
            cy.postMessageAs({sender: admin, message: 'Now from admin', channelId: channel.id});
```

Doing so, you'll have something like "September 20" date line/separator.
![Screen Shot 2020-09-30 at 9 01 38 PM](https://user-images.githubusercontent.com/5334504/94688426-2e17fd80-0360-11eb-89d5-8b5328de0445.png)

Reply:
Thanks!